### PR TITLE
[IGA] Delete nurbs_application

### PR DESCRIPTION
**Pull request by oberbichler**
It seems that nobody is maintaining this application anymore. Should we keep the geometry object inside the core?

### CODE REVIEW

**Comment by tteschemacher** [comment]
I would delete it to avoid confusions. I am anyways not sure whether it is working.

File: applications/nurbs_application/custom_elements/nurbs_poisson_2d.cpp
Comment (loumalouomega) [inline applications/nurbs_application/custom_elements/nurbs_poisson_2d.cpp:3]: I like the author

**Review by loumalouomega — APPROVED** [review]
By by nurbs application, this give memories, bad memories, but memories

**Comment by philbucher** [comment]
guys pls wait for @RiccardoRossi to give approval
He is on vacation currently

File: applications/nurbs_application/custom_elements/nurbs_poisson_2d.cpp
Comment (tteschemacher) [reply applications/nurbs_application/custom_elements/nurbs_poisson_2d.cpp:3]: We all know that this is not true

**Pull request merged**
